app/services/invoice_pdf_generator.rb

```diff
@@ -46,15 +46,15 @@ class InvoicePdfGenerator
   # - :client_address (String) - Address of the client
   # - :items (Array of Hashes) - List of items, each item being a hash with keys:
   #   - :item_name (String) - Name of the item
-  #   - :price_cents (Integer) - Price of the item in cents
+  #   - :price (Money) - Price of the item as a Money object
```

Comment:
`items[*].price` is not actually a Money object, it can also be a hash with `cents` and `currency_iso`. Maybe it would be better to parse and validate the input data during initialization (call `to_money` on all values and replace in the input hash) instead of letting the rest of the code (forget to) call it several times?